app/experimenter/experiments/tests/factories/nimbus.py

```diff
@@ -66,17 +67,23 @@ class Meta:
 
     @factory.post_generation
     def probe_sets(self, create, extracted, **kwargs):
+        is_primary = kwargs.pop("is_primary", True)
         if not create:
             # Simple build, do nothing.
             return
 
-        if extracted:
+        if isinstance(extracted, Iterable):
```

Comment:
Yeah this works, I was also thinking it could be 

`extracted=None`
`if extracted is not None:`

Reply:
Since the code here relies on having an iterable with the for loop, this seemed like the best option. It would error if this branch executed with a `True` for example (though I can't imagine why one would pass that in).